dlp/src/main/java/dlp/snippets/InfoTypesList.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dlp.snippets;
+
+// [START dlp_list_info_types]
+
+import com.google.cloud.dlp.v2.DlpServiceClient;
+import com.google.privacy.dlp.v2.InfoTypeDescription;
+import com.google.privacy.dlp.v2.ListInfoTypesRequest;
+import com.google.privacy.dlp.v2.ListInfoTypesResponse;
+
+import java.io.IOException;
+import java.util.List;
+
+public class InfoTypesList  {
+
+    public static void listInfoTypes() throws IOException {
+        // TODO(developer): Replace these variables before running the sample.
+        String filter = "supported_by=INSPECT";
+        String languageCode= "en-US";
```

Comment:
These two fields (`filter` and `languageCode`) aren't required that the user set to run, so we should inline them in the sample below instead.

Reply:
We should also add links to the docs for these resources, so users know other options that are available for them.

Reply:
I inlined them and added comments to provide context on other options, but couldn't really find a good link for more info on filters. I got the supported filter types from here: https://cloud.google.com/dlp/docs/reference/rpc/google.privacy.dlp.v2#google.privacy.dlp.v2.InfoTypeSupportedBy